lib/staticAssetLoader/README.md

```diff
@@ -0,0 +1,109 @@
+# Static Asset Loader
+
+This can be used by client-side web applications to request and receive the
+static resources they require from a suitable static asset server. The
+primary objective is to allow multiple discrete web applications on the same
+page to converge on shared dependencies. Each dependency is only loaded
+once and this reduces the overall page footprint.
+
+## Basic Usage
+
+### Configuration
+
+The static asset loader must be configured with the following:
+
+  * A function to generate asset URLs from asset name and version strings.
+  * The name of the application namespace.
+
+Applications sharing static assets must use the same namespace.
+
+```javascript
+var namespacer = require('bv-ui-core/lib/namespacer');
```

Comment:
Since the import of `namespacer` is shown, should it also be used in this example?